README.md

```diff
@@ -493,11 +496,17 @@ files before adding the `-exec` part.
 
 ## Running as a daemon
 
-To speed up importing, import-js can run as a daemon. Start the daemon by
-running `importjsd`. The daemon accepts commands sent via `stdin`. Each command
-is a (oneline) JSON string ending with a newline. The command structure is
-basically the same as for the command-line tool, but wrapped in JSON instead of
-expressed on the command line. Here are a few examples:
+Instead of invoking the `importjs` command-line tool every time you import
+something, you can run ImportJS in a background process and communicate with
+it using `stdin` and `stdout`. This will make importing faster because we don't
+have to spin up a node environment on every invocation. Chances are your editor
+plugin is already using the daemon for importing.
```

Comment:
It might be a good idea to be more clear about how if you are using one of the official editor plugins, that you won't need to do anything to get these benefits. This section is intended for advanced usage only.